### PR TITLE
Feature 35 mentor acc max

### PR DESCRIPTION
**Pull request by maxzhovtyj**

### CODE REVIEW

**Review by webdevelops — APPROVED** [review]

File: src/app/pages/account/components/account-mentor/account-mentor.component.html [hunk 1]
```diff
@@ -1 +1,185 @@
...
+            rows="5"
+            class="mentor__control"
+            formControlName="about"
+          >
+          </textarea>
+        </div>
+      </div>
+
+      <div class="mentor__form-field">
+        <span class="mentor__label mentor__teach">A can teach</span>
```
Comment (savchuknatalia) [inline src/app/pages/account/components/account-mentor/account-mentor.component.html]: grammar error: "A can teach"

File: src/app/pages/account/components/account-mentor/account-mentor.component.html [hunk 2]
```diff
@@ -1 +1,185 @@
...
+          <input
+            type="text"
+            formControlName="facebook"
+            class="mentor__control"
+          />
+        </div>
+      </div>
+
+      <div class="mentor__form-field">
+        <span class="mentor__label">Youtube</span>
```
Comment (savchuknatalia) [inline src/app/pages/account/components/account-mentor/account-mentor.component.html]: "YouTube" will be more YouTube

File: src/app/pages/account/components/account-mentor/account-mentor.component.html [hunk 3]
```diff
@@ -1 +1,185 @@
...
+            rows="5"
+            class="mentor__control"
+            formControlName="certificates"
+          >
+          </textarea>
+        </div>
+      </div>
+
+      <div class="mentor__form-field">
+        <span class="mentor__label">I can work with group</span>
```
Comment (savchuknatalia) [inline src/app/pages/account/components/account-mentor/account-mentor.component.html]: "I can work with a group" is correct

File: src/app/pages/account/components/account-mentor/account-mentor.component.html (hunk 1 above)
Comment (maxzhovtyj) [reply src/app/pages/account/components/account-mentor/account-mentor.component.html]: complete

File: src/app/pages/account/components/account-mentor/account-mentor.component.html (hunk 2 above)
Comment (maxzhovtyj) [reply src/app/pages/account/components/account-mentor/account-mentor.component.html]: complete

File: src/app/pages/account/components/account-mentor/account-mentor.component.html (hunk 3 above)
Comment (maxzhovtyj) [reply src/app/pages/account/components/account-mentor/account-mentor.component.html]: complete

**Review by savchuknatalia — APPROVED** [review]

**Review by farmProw — APPROVED** [review]

**Pull request merged**